ui-tests/src/test/java/org/openmrs/reference/page/VisitNotePage.java

```diff
@@ -55,6 +62,17 @@ public void addNote(String note) {
         findElement(NOTE).clear();
         setText(NOTE, note);
     }
+    public void editVisitNote() {
+        findElement(EDIT_VISIT_NOTE).click();
+    }
+    
+    public void viewVisitNote() {
+        findElement(VIEW_VISIT_NOTE).click();
+    }
+    
+    public void editButton() {
+        findElement(EDIT_BUTTON).click();
```

Comment:
this should be dropped, it's unused and doesn't sound a right action to qualify for a method

Reply:
True we can get rid of it since it does the same functionality with **EDIT_VISIT_NOTE**

Reply:
Resolved thanks